test/Microsoft.AspNetCore.Mvc.Core.Test/Internal/ControllerBinderDelegateProviderTest.cs

```diff
@@ -49,10 +49,21 @@ public class ControllerBinderDelegateProviderTest
                 .Setup(b => b.BindModelAsync(It.IsAny<DefaultModelBindingContext>()))
                 .Verifiable();
 
-            var mockValidator = CreateMockValidator();
+            var mockValidator = new Mock<IModelValidator>(MockBehavior.Strict);
+            mockValidator
+                .Setup(o => o.Validate(
+                    It.IsAny<ModelValidationContext>()));
```

Comment:
These three lines should fit on one and will be easier to read as one. Do the same for this particular `mockValidator` setup pattern later in the tests. (Otherwise, whitespace looks great.)